app/presenters/tree_node/menu/item.rb

```diff
@@ -0,0 +1,75 @@
+module TreeNode
+  module Menu
+    class Item < Node
+      set_attribute(:key) { "#{@options[:node_id_prefix]}__#{@object.feature}" }
+
+      set_attribute(:title) { _(details[:name]) }
+
+      set_attribute(:tooltip) { _(details[:description]) || _(details[:name]) }
+
+      set_attribute(:image, "100/feature_node")
+
+      set_attribute(:checkable) { @options[:editable] }
+
+      set_attribute(:selected) do
+        self_selected? || select_by_kids_selected
+      end
+
+      set_attribute(:no_click, true)
+
+      set_attribute(:hide_checkbox, false)
```

Comment:
Isn't this default? I mean if there are checkboxes, they're displayed if not said othetwise.